com.shatteredpixel.shatteredpixeldungeon.appdata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.shatteredpixel.shatteredpixeldungeon</id>
+	<name>Shattered Pixel Dungeon</name>
+	<summary>Traditional roguelike game with pixel-art graphics and simple interface</summary>
+	<metadata_license>GPL-3.0-only</metadata_license>
```

Comment:
We prefer CC0, but not sure what the license should be if description is taken from Play Store.

Reply:
@00-Evan Would you be willing to license the game description under CC0? Here's more info on AppData: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

And are you interested in having push-access to this repo? More info on Flatpak: https://flatpak.org/

Reply:
Thanks for the mention. I have no interest in distributing the desktop libgdx build as it is not complete and will likely become deprecated in the future, as I want to eventually move the android-only code to a proper unified codebase.

Naturally, as the code is GPLv3, you are welcome to put shattered-gdx on flatpak if you like.

In the future, when I am ready to officially support the desktop distribution of the game my stance on this will likely change.

Reply:
Thanks for the response. What's your stance on licensing the description under CC0?